File: INICIANTE/1035/main.dart
import 'dart:io';

void main() {
  var [a, b, c, d] = stdin.readLineSync()!.split(' ').map(int.parse).toList();
  bool accepted = b > c && d > a && c + d > a + b && c >= 0 && d >= 0 && a % 2 == 0;
  print(accepted ? 'Valores aceitos' : 'Valores nao aceitos');
}

File: INICIANTE/1035/main.js
let [a, b, c, d] = require('fs').readFileSync('/dev/stdin', 'utf-8').split(' ').map(Number);
let accepted = b > c && d > a && c + d > a + b && c >= 0 && d >= 0 && a % 2 === 0;
console.log(accepted ? 'Valores aceitos' : 'Valores nao aceitos');
